lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -474,10 +499,17 @@ private JsonObject getResponseBody(String url, Dependency artifactToSearch) thro
 				}
 			} else {
 				LOGGER.log(Level.SEVERE,
-						"Maven Central Repo search failed for " + String.join(":", artifactToSearch.getGroupId(),
-								artifactToSearch.getArtifactId(), artifactToSearch.getVersion()),
-						response.getStatusLine().getReasonPhrase());
+						"Maven Central Repo search failed for " //
+								+ String.join(":", artifactToSearch.getGroupId(),
+										artifactToSearch.getArtifactId(), artifactToSearch.getVersion())
+								+ ": Status code received: " + response.statusCode());
 			}
+		} catch (IOException | SecurityException e) {
```

Comment:
might as well catch Exception, in case there's a parsing issue

Reply:
I thought about using it, but I don't think it's a good idea to log every exception  - f.i. I'd like to exclude timeouts. Unfortunately  timeouts are reported as IOException,   so probablly you're right.

Reply:
Changed to `catch (Exception e) {`